src/schematools/contrib/django/models.py

```diff
@@ -379,6 +385,7 @@ class DatasetTable(models.Model):
 
     dataset = models.ForeignKey(Dataset, on_delete=models.CASCADE, related_name="tables")
     name = models.CharField(max_length=100)
+    version = models.TextField(default=SemVer("1.0.0"))
```

Comment:
Ik dacht dat in de datasets_dataset en datasets_datasettable tabellen alleen kolommen stonden die dso nodig had. Het zijn toch geen directe representaties van amsterdam-schema objecten?

Reply:
It is similar to what Nick did when [he added dataset versioning](https://github.com/Amsterdam/schema-tools/commit/32a47f1fdf50618ea23820afee0f1b88d2f41a66#diff-1b59f231374bf5878b1793280aa14481447515ba99e7f39c5bfeeb6e446898acR199)